packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/components/TransactionsCompletedRowAccountDetails/TransactionsCompletedRowAccountDetails.tsx

```diff
@@ -25,7 +28,9 @@ const TransactionsCompletedRowAccountDetails: React.FC<TProps> = ({
     isDemo,
     isInterWallet = false,
     mt5Group,
+    transactionID,
 }) => {
+    const { isMobile } = useDevice();
```

Comment:
instead of `isMobile` can we use `!isDesktop` for tablet support

Reply:
🙄